lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -61,6 +62,18 @@ def repository_id(options)
       find_resource(:repositories, options)['id']
     end
 
+    def repository_ids(options)
+      return options['option_repository_ids'] unless options['option_repository_ids'].nil?
+
+      key_names = HammerCLI.option_accessor_name 'names'
+      key_product_id = HammerCLI.option_accessor_name 'product_id'
+      options[key_product_id] ||= product_id(scoped_options 'product', options)
+      find_resources(:repositories, options)
+        .select { |repo| options[key_names].include? repo['name'] }.map { |repo| repo['id'] }
+      # ^ This could be removed if Katello's repositories controller allowed to filter
+      #   repositories index by repository names
```

Comment:
I tend towards not including this comment, this is a general issue with how tenancy works in Katello and I don't want someone reading this thinking it is a simple change to the repositories controller. Right now filtering by name can occur via scoped_search or if we combined name + product to scope things properly.